src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/ToModelTransformers.kt

```
@@ -464,3 +467,26 @@ fun CaseloadResult.transformToUnstartedRecord(
     isOnProbation = false,
   )
 }
+
+fun Licence.getUpdatedByFullName(): String {
+  val staffMember = this.updatedBy
+  return if (staffMember != null) {
+    "${staffMember.firstName} ${staffMember.lastName}"
+  } else {
+    ""
+  }
+}
+
+fun Licence.getSubmittedByFullName(): String {
+  val staffMember = when (this) {
+    is HardStopLicence -> this.submittedBy
+    is CrdLicence -> this.submittedBy
+    is VariationLicence -> this.submittedBy
+    else -> null
```

maybe better `error("some reason")` here, so we fail loudly.